tools/benchmark/src/mocha/memoryTestRunner.ts

```diff
@@ -116,6 +116,11 @@ export interface MemoryTestObjectProps extends MochaExclusiveOptions, Titled, Be
 	 * Use a lower number to drop the highest/lowest measurements.
 	 */
 	samplePercentageToUse?: number;
+
+	/**
+	 * The baseline memory usage to compare against for the test, which is used to determine if the test regressed.
+	 */
+	baselineMemoryUsage?: number;
```

Comment:
readonly?

Reply:
Cool, I didn't know you could specify that in the interface